### ISSUE TITLE
[Bug] No module named 'TTS.trainer'

### ISSUE THREAD

**Issue by FreeSlave** [coqui-ai/TTS#625]
I'm trying to follow [this guide](https://tts.readthedocs.io/en/latest/tutorial_for_nervous_beginners.html) but it has given me an import error

```
Traceback (most recent call last):
  File "train.py", line 12, in <module>
    from TTS.trainer import init_training, Trainer, TrainingArgs
ModuleNotFoundError: No module named 'TTS.trainer'
```

I see that there're different versions of the software and the api probably has been changed since this guide was written. Can the tutorial be updated to match the current `main` branch? What release should I use to follow the tutorial?

**Comment by erogol** [comment]
giving you an import error as you do what? 

Please post issues respecting to the issue template. There is a template for a reason .

If you pip installed it, git clone and install locally. Pip is only for inference.

**Comment by FreeSlave** [comment]
@erogol I'm trying to run an example of Training a tts Model in a Pure Python Way ([the code snippet from the docs](https://tts.readthedocs.io/en/latest/tutorial_for_nervous_beginners.html#pure-python-way))
I've cloned the repo and setup and activated the python virtual environment. I ran the `make install`. Then if I run `pip install TTS` it says 
```
Requirement already satisfied: TTS in /home/freeslave/git_projects/TTS (0.0.15.1)
```

Also checking the path of the TTS package in python:
```
>>> import TTS
>>> print(TTS.__file__)
/home/freeslave/git_projects/TTS/TTS/__init__.py
```

So I guess the installation is correct?

I've copied the code from the tutorial, created a file outside of repo tree (if python can find a module then it should not be an issue) and run as usual `python filename.py`

Note that it can import `TTS.tts.configs` and fails only on importing `TTS.trainer`

**Comment by FreeSlave** [comment]
Ok, I can see the pip reports the wrong version (0.0.15.1 instead of 0.1.2). I've cloned the repo before you released the 0.1.0 and never pulled.
Sorry for inconvinience.

**Comment by siagupta-03** [comment]
> @erogol I'm trying to run an example of Training a tts Model in a Pure Python Way ([the code snippet from the docs](https://tts.readthedocs.io/en/latest/tutorial_for_nervous_beginners.html#pure-python-way)) I've cloned the repo and setup and activated the python virtual environment. I ran the `make install`. Then if I run `pip install TTS` it says
> 
> ```
> Requirement already satisfied: TTS in /home/freeslave/git_projects/TTS (0.0.15.1)
> ```
> 
> Also checking the path of the TTS package in python:
> 
> ```
> >>> import TTS
> >>> print(TTS.__file__)
> /home/freeslave/git_projects/TTS/TTS/__init__.py
> ```
> 
> So I guess the installation is correct?
> 
> I've copied the code from the tutorial, created a file outside of repo tree (if python can find a module then it should not be an issue) and run as usual `python filename.py`
> 
> Note that it can import `TTS.tts.configs` and fails only on importing `TTS.trainer`

facing the same error pls help
